playground/src/components/custom-views/custom-panel-demo.jsx

```diff
@@ -1,15 +1,45 @@
-import { useState } from 'react';
+import { useMemo, useState } from 'react';
 import {
-  CustomPanel,
+  CustomViewLoader,
   useModalState,
 } from '@commercetools-frontend/application-components';
 import SecondaryButton from '@commercetools-uikit/secondary-button';
 import Spacings from '@commercetools-uikit/spacings';
 import Text from '@commercetools-uikit/text';
 
+export const CUSTOM_VIEW_ID = '290f83df-d86d-417c-ab24-41697e33483c';
+
+const DEMO_CUSTOM_VIEW = {
+  id: CUSTOM_VIEW_ID,
+  defaultLabel: 'Demo Custom View',
+  labelAllLocales: {},
+  url: `http://localhost:3001/custom-view/${CUSTOM_VIEW_ID}`,
+  type: 'CustomPanel',
+  typeConfig: {
+    size: 'SMALL',
+  },
+  locators: ['/almond-40/app-kit-playground/custom-panel'],
+  permissions: [
+    {
+      name: 'ViewProducts',
+      oAuthScopes: ['view_channels'],
+    },
+  ],
+};
+
 function CustomPanelDemo() {
```

Comment:
This component just renders a couple of buttons to open a Custom View (with different sizes).